sdk/src/main/java/io/dapr/client/domain/ConfigurationItem.java

```diff
@@ -20,10 +20,17 @@
  * A configuration item from Dapr's configuration store.
  */
 public class ConfigurationItem {
-  private final String key;
-  private final String value;
-  private final String version;
-  private final Map<String, String> metadata;
+  private String key;

```

Comment:
why is the final modifier removed?

Reply:
This is to make jackson library able to parse the Request body to `SubscribeConfigurationResponse ` sent by dapr to app endpoint in sdk/examples. Jackson needed the `no Arg constructor` and then since we are not actually doing any setting/ initialisation, it was complaining to initialise the final variables.

Reply:
Another option is here to use `@JsonCreator` annotation ...

Reply:
I agree with Mukundan. This also should make use of the constructor we have.

Reply:
done